.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: python
+python:
+- '3.6.3'
+before_script:
+- pip install pybuilder
+script: pyb package publish
```

Comment:
Does this run tests?

Reply:
Yes, by default unit tests are always run. I haven't done anything with integration test yet, so can't say anything about that. Also another thing in mind is that, currently there is no way to manually skip unit tests using some command line, like we can do in Maven.